store/pruning/manager.go

```diff
@@ -177,7 +217,7 @@ func loadPruningSnapshotHeights(db dbm.DB) ([]int64, error) {
 	return pruneSnapshotHeights, nil
 }
 
-func int64SliceToBytes(slice []int64) []byte {
+func int64SliceToBytes(slice ...int64) []byte {
```

Comment:
why did this need to be changed?

Reply:
I am storing the max height now instead of the full list.  There will not be any in-flight snapshots after a restart.